### PR TITLE
Create MVP architecture

### PR DESCRIPTION
**Pull request by yusufbayam**
Implemented MVP architecture and added necessary dependencies

### CODE REVIEW

File: android/app/src/main/java/com/bounswe2020group3/paperlayer/RegisterFragment.kt [hunk 1]
```diff
@@ -1,58 +1,20 @@
 package com.bounswe2020group3.paperlayer
 
+import android.app.Fragment
```
Comment (ybedirhanpak) [inline android/app/src/main/java/com/bounswe2020group3/paperlayer/RegisterFragment.kt:3]: Why using native Fragments ? AFAIK native Fragments are not recommended to use. Instead we should use either androidx fragment or support fragment.
https://www.andreasschrade.com/android-native-fragment-or-support-fragment-or-androidx-fragment

File: android/.idea/misc.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="JDK" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```
Comment (ybedirhanpak) [inline android/.idea/misc.xml:3]: This is a general question for changes under .idea folder. Did you update them by hand or by running some command? I'm trying to understand why these changes happened and needed.

File: android/app/src/main/java/com/bounswe2020group3/paperlayer/RegisterFragment.kt (hunk 1 above)
Comment (yusufbayam) [reply android/app/src/main/java/com/bounswe2020group3/paperlayer/RegisterFragment.kt:3]: These fragments were implemented by Ramazan and I did not change them. We can migrate to androidx and there wont be any problems since the keywords are identical for native and androidx library.

File: android/.idea/misc.xml (hunk 2 above)
Comment (yusufbayam) [reply android/.idea/misc.xml:3]: They are updated by Gradle automatically after the changes i made on the project. Here it is specifying JDK version to avoid any compatibilty problems in the future.

Comment (ybedirhanpak) [reply android/.idea/misc.xml:3]: Thanks

File: android/app/src/main/java/com/bounswe2020group3/paperlayer/RegisterFragment.kt (hunk 1 above)
Comment (ybedirhanpak) [reply android/app/src/main/java/com/bounswe2020group3/paperlayer/RegisterFragment.kt:3]: All right, this is not a blocker for this pull request but we should be aware of it.

**Review by ybedirhanpak — APPROVED** [review]

**Review by rambo250 — APPROVED** [review]
